<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolverWidget</class>
 <widget class="QWidget" name="SolverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Solve a problem</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="pathPlannerLabel">
          <property name="text">
           <string>Path planner</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="pathPlannerComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="pathProjectorLabel">
          <property name="text">
           <string>Path projector</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="pathProjectorComboBox"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="pathOptimizerLabel">
          <property name="text">
           <string>Path optimizer</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="pathOptimizerButton">
          <property name="text">
           <string>Select path optimizers</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="pathValidationLabel">
          <property name="text">
           <string>Path Validation</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="pathValidationComboBox"/>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="pathValidationPenetration"/>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="pathProjectorDiscontinuity"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="pathProjectorDiscontinuityLabel">
          <property name="text">
           <string>Allowed Discontinuity</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="pathValidationPenetrationLabel">
          <property name="text">
           <string>Allowed Penetration</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Steering Method</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="steeringMethodComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSolve">
          <property name="text">
           <string>&amp;Solve</string>
          </property>
          <property name="icon">
           <iconset theme="games-solve">
            <normaloff>../../include/hpp/gui/ui</normaloff>../../include/hpp/gui/ui</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSolveAndDisplay">
          <property name="text">
           <string>Solve and display</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="optimizeButton">
          <property name="text">
           <string>Optimize Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonInterrupt">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>&amp;Interrupt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Roadmap</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loadRoadmap">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveRoadmap">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
